python.py

```diff
@@ -0,0 +1,3 @@
+def square(x):
+""" This function will square any number"""
+    return x**2
```

Comment:
What if `x` is not a number? can you please consider that option?

Reply:
I think that is beyond the scope of this function.